utilities/blob_db/blob_db_impl.cc

```diff
@@ -1042,6 +1036,18 @@ Status BlobDBImpl::PutWithTTL(const WriteOptions& options,
           : -1);
 }
 
+Slice BlobDBImpl::GetCompressedSlice(const Slice& raw,
+                                     std::string* compression_output) const {
```

Comment:
`compression_output` is needed. It backs the return Slice, and have to be destroy after we finish using the value. I need to fix compile error though..